picoscope/picobase.py

```diff
@@ -335,6 +335,20 @@ def getMaxMemorySegments(self):
         segments = self._lowLevelGetMaxSegments()
         return segments
 
+    def setExtTriggerRange(self, VRange=0.5):
```

Comment:
Inconsistent default value with VRangeAPI.

I don't think a default value makes sense for this function.

What you did below is fine.